<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GexDbPlugin_ItemSelectionDialog_Base</class>
 <widget class="QDialog" name="GexDbPlugin_ItemSelectionDialog_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database item selection</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>145</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
